packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -658,7 +657,6 @@ export const NewUserNotification = registerNotificationType({
 export const NewMessageNotification = registerNotificationType({
   name: "newMessage",
   userSettingField: "notificationPrivateMessage",
-  allowedChannels: ["onsite", "email", "both"],
```

Comment:
I have removed the ability to require notifications to be turned on (see [slack discussion](https://cea-core.slack.com/archives/CJY3ZKP62/p1740161585169789?thread_ts=1740145651.444799&cid=CJY3ZKP62)), so this is no longer needed